schoolopy/main.py

```diff
@@ -297,18 +297,13 @@ def get_course_sections(self, course_id=None, include_past=False):
         """
         return [Section(raw) for raw in self._get('courses/%s/sections' % course_id)['section']]
 
-    def get_sections(self, course_id=None):
+    def get_sections(self):
         """
         Get data on all sections in which a user is enrolled.
-        If course_id is passed, the call will be passed through to get_course_sections,
-        which performs the original purpose of this method.
 
         :return: List of Section objects.
         """
-        # For backwards-compatibility
-        if course_id is not None:
-            return self.get_course_sections(course_id)
-        return [Section(raw) for raw in self._get('sections')['section']]
+        return [Section(raw) for raw in self._get(f"users/{self.get_me()['uid']}/sections")['section']]
```

Comment:
Could you add a parameter to pass in a custom user ID to fetch, defaulting to `None`, and if it's `None` then fall back to the `get_me` `uid`?

Reply:
Just because there's nowhere else that we do two requests in one method, and people might already know the user ID they want and not want to do a repeated API query

Reply:
> Could you add a parameter to pass in a custom user ID to fetch, defaulting to `None`, and if it's `None` then fall back to the `get_me` `uid`?

for sure, i just forgot that was something you could do lol